### PR TITLE
[patch] upgrade closure-compiler to latest

### PR DESCRIPTION
**Pull request by branch-nchoudry**
## Description
The current version of google-closure-compiler `20220601.0.0` is outdated and vulnerable to CVE-2022-25647. 

Change release note - https://github.com/google/closure-compiler/releases/tag/v20230206

Fixes # (issue)
Upgrade google-closure-compiler to `20230206.0.0` to patch CVE-2022-25647.

google-closure-deps: `^20220502.0.0` to `^20230103.0.0` and google-closure-library: `^20220502.0.0` to `^20230103.0.0`

## Type of change

Please delete options that are not relevant. - 
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Pre-commit lint checks
- [ ] Unit test - TBD
- [ ] Integration test - TBD

## JS Budget Check

Please mention the size in kb before abd after this PR - TBD

| Files            | Before      | After       | 
| -----------      | ----------- | ----------- |
| dist/build.js.   |    164KB         |      160KB       |
| dist/build.min.js|      74KB       |      74KB       |

## Checklist:
**TBD**
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Mentions: 
List the person or team responsible for reviewing proposed changes.

cc @BranchMetrics/saas-sdk-devs for visibility.

### CODE REVIEW

**Comment by branch-nchoudry** [comment]
FYI - Request from `firstcard.app`

**Review by JagadeeshKaricherla-branch — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
 		"sinon": "1.17.3"
 	},
 	"dependencies": {
-		"google-closure-compiler": "^20220601.0.0",
+		"google-closure-compiler": "^20230206.0.0",
 		"google-closure-deps": "^20220502.0.0",
```
[the remark this answers is not in the document]
Comment (branch-nchoudry) [reply package.json]: Done, please review!

**Comment by JagadeeshKaricherla-branch** [comment]
LGTM

**Comment by dmelo** [comment]
@branch-nchoudry , do you know when this is planned to be released?

**Comment by branch-nchoudry** [comment]
> @branch-nchoudry , do you know when this is planned to be released?

We will release these changes next week. Thanks!

**Comment by dmelo** [comment]
@branch-nchoudry , any update on publishing this PR?

**Comment by dmelo** [comment]
Hi @branch-nchoudry. Do you have any updates?

**Comment by JagadeeshKaricherla-branch** [comment]
We are currently testing these changes and will make a release post testing.

**Comment by dmelo** [comment]
Hi @JagadeeshKaricherla-branch and @branch-nchoudry , any updates about when this will be released?

**Comment by JagadeeshKaricherla-branch** [comment]
This is included in our latest release.

**Pull request merged**